<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consultarCarteirinha</class>
 <widget class="QMainWindow" name="consultarCarteirinha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(167, 167, 167);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF do aluno ou c√≥digo da carteirinha:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entradaCPF">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(209, 209, 209);</string>
    </property>
    <widget class="QLabel" name="labelNome">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCurso">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Curso</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAno">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ano Inicio</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCodigo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Codigo Carteirinha</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelValidade">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Validade</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInstituto">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Inst</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnConsultar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVoltar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMensagem">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
